### PR TITLE
Create local users based on user-specified mapfiles (SOFTWARE-3960)

### PR DESCRIPTION
**Pull request by brianhlin**
- Treat local and remote users as the same
- Assume a single remote host. This allows us to remove `foreach_bosco_endpoint` and related scripts since it's easy to parse the local/remote users from the mapfiles.
- Requires https://github.com/slateci/slate-catalog/pull/219

### CODE REVIEW

File: 30-remote-site-setup.sh [hunk 1]
```diff
@@ -1,60 +1,52 @@
 #!/bin/bash
 
-. /usr/local/bin/foreach_bosco_endpoint.sh
-
 BOSCO_KEY=/etc/osg/bosco.key
+REMOTE_HOST_KEY=`ssh-keyscan -H "$REMOTE_HOST"`
```
Comment (edquist) [inline 30-remote-site-setup.sh]: Where does `REMOTE_HOST` getting set?

**Comment by brianhlin** [comment]
From the environment, see https://github.com/slateci/slate-catalog/pull/219

File: 30-remote-site-setup.sh (hunk 1 above)
Comment (brianhlin) [reply 30-remote-site-setup.sh]: From the environment, see slateci/slate-catalog#219

Comment (brianhlin) [reply 30-remote-site-setup.sh]: @edquist ping

Comment (edquist) [reply 30-remote-site-setup.sh]: Alright.  It wasn't super clear in the scripts here what was expected to be set in the script vs from outside (perhaps a comment somewhere about the expected externally-set environment might be useful), but in any case sounds like it should work.

**Pull request merged**